### PR TITLE
Fix issue#109 - add fsm models into Django admin UI to allow to edit FSM's

### PR DESCRIPTION
**Pull request by maxsocl**
@cjlee112 @VladimirFilonov 

Changes:
- add blank=True to several fields to allow to correctly edit/create models
- add `__unicode__` method to several FSM models to create clear representations for models

Issue https://github.com/cjlee112/socraticqs2/issues/109

### CODE REVIEW

File: mysite/fsm/models.py [hunk 1]
```diff
@@ -152,9 +152,9 @@ class FSM(models.Model):
     """
     name = models.CharField(max_length=64, unique=True)
     title = models.CharField(max_length=200)
-    description = models.TextField(null=True)
-    help = models.TextField(null=True)
-    startNode = models.ForeignKey('FSMNode', related_name='+', null=True)
+    description = models.TextField(null=True, blank=True)
+    help = models.TextField(null=True, blank=True)
```
Comment (VladimirFilonov) [inline mysite/fsm/models.py]: help if shadows built-in function help. This is not critical thing =)

File: mysite/fsm/models.py [hunk 2]
```diff
@@ -479,6 +492,9 @@ def find_live_sessions(cls, user):
             isLiveSession=True, activity__course__role__user=user
         )
 
+    def __unicode__(self):
+        return '::'.join((self.user.username, str(self.fsmNode)))
```
Comment (VladimirFilonov) [inline mysite/fsm/models.py]: it's better to return unicode string here. u'::'

Comment (maxsocl) [inline mysite/fsm/models.py]: Yes, sure. Unicode should return unicode.

File: mysite/fsm/models.py (hunk 1 above)
Comment (maxsocl) [inline mysite/fsm/models.py]: Yes, but it is an old code and this field is used a lot in project - I think it is a refactoring task and in current task we should not do this.
There are a lot of things in the core application to refactor.

**Comment by sengottuvelk** [comment]
@maxsocl 

In `mysite/fsm/models.py`, default values for `DateTimeField ` is set using `django.utils.timezone`.

Django's `DateTimeField` itself provides `auto_now_add` option, where it uses the same [django's timezone utils](https://github.com/django/django/blob/1.7/django/db/models/fields/__init__.py#L1257). 

Not a major one. Can be considered for re-factoring.

**Comment by sengottuvelk** [comment]
LGTM, Merge when ready.

**Comment by maxsocl** [comment]
> @maxsocl
> 
> In mysite/fsm/models.py, default values for DateTimeField is set using django.utils.timezone.
> 
> Django's DateTimeField itself provides auto_now_add option, where it uses the same django's timezone utils.
> 
> Not a major one. Can be considered for re-factoring.

Yes - I planned to change this on next refactor step.

**Pull request merged**